include/macis/gf/gf.hpp

```diff
@@ -244,10 +229,10 @@ void get_GF_basis_AS_1El(int orb, bool sp_up, bool is_part, const VectorXd &wfn,
   std::bitset<nbits> uni_string;
   std::vector<std::bitset<nbits>> founddets;
   founddets.reserve(trunc_size);
-  std::map<std::bitset<nbits>, size_t, BitSetComparator<nbits>> founddet_pos,
-      basedet_pos;
+  std::map<std::bitset<nbits>, size_t, bitset_less_comparator<nbits>>
+      founddet_pos, basedet_pos;
   typename std::map<std::bitset<nbits>, size_t,
```

Comment:
This is a great example of where it might be good to make an alias for readability. e.g.
```
using bitset_map = std::map< std::bitset<nbits>, size_t, bitset_less_comparator<nbits>>;
using bitset_map_iterator = typename bitset_map::iterator;
```